sql/migrations/warehouse_always/000001_alter_jsonb_to_text.up.tmpl

```diff
@@ -9,24 +9,24 @@
       SELECT data_type INTO wh_schemas_data_type
       FROM information_schema.columns
       WHERE table_name = 'wh_schemas' AND column_name = 'schema';
-      
+
       SELECT data_type INTO wh_staging_files_data_type
       FROM information_schema.columns
       WHERE table_name = 'wh_staging_files' AND column_name = 'schema';
-      
+
       SELECT data_type INTO wh_uploads_data_type
       FROM information_schema.columns
       WHERE table_name = 'wh_uploads' AND column_name = 'schema';
-      
+
       -- Handle wh_schemas table
-      IF wh_schemas_data_type != 'text' THEN
-        RAISE NOTICE 'Altering column "schema" in table "wh_schemas" from % to text', wh_schemas_data_type;
-        ALTER TABLE wh_schemas ALTER COLUMN schema TYPE TEXT USING schema::TEXT;
-        RAISE NOTICE 'Column "schema" in table "wh_schemas" successfully altered to text.';
+      IF wh_schemas_data_type != 'jsonb' THEN
```

Comment:
Should we move these "warehouse_always" migrations to the normal migrations list?